### ISSUE TITLE
Error Code -2147467259

### ISSUE THREAD

**Issue by keebou** [ronancpl/HeavenMS#554]
Using the rev203 source.
Been running into this error and can't really understand what I am doing wrong to be getting this. Following the Readme file, I place the commit203 wz's into the v83 folder and when i place the recommended localhost, I get the wz mistmatch. I tried looking into the closed cases here, but they do not have their solutions in it. Any ideas why this is happening? I have not changed the host IP. The connection is going through and I can see it in the bat file. ![image](https://user-images.githubusercontent.com/59017995/71087376-e289bf80-2150-11ea-845b-35cebc3f3d75.png)

**Comment by ronancpl** [comment]
This seems to be a graphics driver issue. More info can be found on:

https://nexoncommunitytech.fandom.com/wiki/Error_code_:_-2147467259_(unspecified_error)

**Comment by ronancpl** [comment]
Depending on whether you are running the game on a dedicated graphics card, I'd suggest trying to run the game on the machine's default card. Maybe this helps.

**Comment by BHB88** [comment]
This is a common error that may relate to the monitor's refresh rate (especially 59hz). 60hz reportedly works best with the client.

**Comment by keebou** [comment]
Hello all! Thank you ronan for the direction! I spent some time looking around that forum and i eventually got to a forum post for MapleRoyals. BHB88 was correct with what he said. I had to adjust my monitors refresh rate to 60.2 in my graphics card setting for the game to work.
![image](https://user-images.githubusercontent.com/59017995/71151056-cf75fe80-21e7-11ea-90ff-f9e7e2455f7f.png)
Thank you for the help and for the source! Doing amazing work.
